core/arch/arm/plat-zynqmp/platform_config.h

```diff
@@ -86,6 +86,11 @@
 #error "Unknown platform flavor"
 #endif
 
+#define CSUDMA_BASE		0xFFC80000
+#define CSUDMA_SIZE		0x1000
```

Comment:
Found following from hardware handoff:
`APERTURE NAME: CSUDMA, START ADDRESS: FFC80000, END ADDRESS: FFC9FFFF`

Which would mean 0x20000 -- which is a bit large space I would say. Don't really see any use for above 0x1000 -- so I suppose here 0x1000 is probably a good value.